fastlane/Fastfile

```diff
@@ -47,11 +47,23 @@ platform :ios do
       files_to_update_without_prerelease_modifiers: files_with_version_number_without_prerelease_modifiers,
       repo_name: repo_name,
       github_rate_limit: options[:github_rate_limit],
-      editor: options[:editor]
+      editor: options[:editor],
+      next_version: options[:next_version]
     )
     compile_autogenerated_header_commit_and_push(options)
   end
 
+  desc "Automatically bumps version, update swift header, edit changelog, and create pull request"
+  lane :automatic_bump do |options|
+    next_version, type_of_bump = determine_next_version_using_labels(
+      repo_name: repo_name,
+      github_rate_limit: options[:github_rate_limit]
+    )
+    options[:next_version] = next_version
+    UI.user_error!('Skipping automatic bump since the next version is a major release') if type_of_bump == :major
```

Comment:
I wonder if this should have been a normal message instead of an error, since this is expected... I guess it's NABD since it shouldn't block anything though.

Reply:
Maybe Danger could show a warning if a PR has `breaking` as a label:

> Warning: this change will require a new major version

Reply:
Well I think this action would be executed from `main` most of the time, not from a PR, so I don't think Danger can warn us easily...

Reply:
I didn't mean in this action, I meant in PRs that have that. So before this action ends up failing, the PR that introduces that breaking change warns us about this happening once merged in `main`. Does that make sense?

Reply:
Ahh yeah, that could be useful. It can be a good reminder 👍